platform/admin/core/admin-core-insecuredefaults/src/main/java/org/codice/ddf/admin/insecure/defaults/service/UsersPropertiesFileValidator.java

```diff
@@ -48,6 +54,10 @@
 
   private String defaultCertificateUserPassword;
 
+  private UserPropertiesDeletionScheduler scheduler;
+
+  static boolean usersPropertiesDeletionFlag;
```

Comment:
✏️ Don't love that this is static. Might be better to make it an instance field and then expose a package-private getter for it so that the value can be accessed in `InsecureDefaultsCollector`.